src/services/codefixes/fixCannotFindModule.ts

```diff
@@ -52,6 +52,6 @@ namespace ts.codefix {
     function getTypesPackageNameToInstall(packageName: string, host: LanguageServiceHost, diagCode: number): string | undefined {
         return diagCode === errorCodeCannotFindModule
             ? (JsTyping.nodeCoreModules.has(packageName) ? "@types/node" : undefined)
-            : (host.isKnownTypesPackageName!(packageName) ? getTypesPackageName(packageName) : undefined); // TODO: GH#18217
+            : (host.isKnownTypesPackageName?.(packageName) ? getTypesPackageName(packageName) : undefined); // TODO: GH#18217

```

Comment:
The comment referring to #18217 can be removed now since the non-null assertion is gone.

```suggestion
            : (host.isKnownTypesPackageName?.(packageName) ? getTypesPackageName(packageName) : undefined);
```

Reply:
Thanks for your suggestion